client/app/main/student/student.controller.js

```diff
@@ -109,6 +109,50 @@ function StudentCtrl($scope, $stateParams, Intervention, Modal, Outreach,
       toastr.error(err);
     });
   };
+  $scope.toggleArchiveOutreach = function(outreach) {
+    var oldValue = outreach.archived;
+    var promise = Outreach.toggleArchive({
+      id: outreach._id
+    }, {
+      archived: !(outreach.archived)
+    }).$promise;
+    promise.then(function(outreach) {
```

Comment:
Using outreach again as the argument name for this anonymous function will clobber the outreach that is used for the argument name in the outside scope. If you use a different name here, it is not only more descriptive but also prevents you from having the alias the outside scope's `outreach` to use it inside the scope of the callback function.